lld/test/ELF/aarch64-jump26-thunk-execute-only.s

```diff
@@ -0,0 +1,22 @@
+// REQUIRES: aarch64
+// RUN: llvm-mc -filetype=obj -triple=aarch64 %S/Inputs/big-execute-only.s -o %tbig.o
+// RUN: llvm-mc -filetype=obj -triple=aarch64 %s -o %t.o
+// RUN: ld.lld %t.o %tbig.o -o %t
+// RUN: llvm-objdump -d --no-show-raw-insn %t | FileCheck %s
+
+.section .text,"axy",@progbits,unique,0
+.globl _start
+_start:
+    b big
+
+// CHECK: Disassembly of section .text:
```

Comment:
Similar comments to before. Can use CHECK-LABEL and remove all but the first address in a sequence.